exercises/gigasecond/Cargo.toml

```diff
@@ -4,4 +4,5 @@ version = "1.1.0"
 
 [dependencies]
 chrono = "0.4"
+time = "0.1"
```

Comment:
The `time` crate has been depreciated. We shouldn't be adding new dependencies to it. Instead, let's use the `chrono` reexport of `Duration`; that way, when that library updates to use `std::time::Duration`, we upgrade with it for free.